### ISSUE TITLE
Cannot use referenced links inside 'notice' blocks

### ISSUE THREAD

**Issue by m-vo** [contao/docs#472]
This is probably a bug in the theme/Hugo I guess?

**Reproducer**

```markdown
{{% notice info %}}
See [the referenced link][Link] for more information.  
{{% /notice %}}

[Link]: https://example.com
```

This will result in a raw `See [the referenced link][Link] for more information.` in the output.

The the link is correctly substituted if not inside a notice block. Also, links are working fine inside the blocks when not using the reference syntax but defining them in place (`[link](https://example.com)`).

**Comment by fritzmg** [comment]
Yeah, unfortunately a known limitation - I think of the shortcode renderer itself, not the the theme. For now you'll have to include the link directly instead of using references.
